src/generated/graphql.ts

```diff
@@ -851,6 +855,10 @@ export type TagsResolvers<ContextType = any, ParentType extends ResolversParentT
   __isTypeOf?: IsTypeOfResolverFn<ParentType, ContextType>;
 };
 
+export interface Int64ScalarConfig extends GraphQLScalarTypeConfig<ResolversTypes['int64'], any> {
```

Comment:
is this used anywhere? I think `new GraphQLScalarType` might be needed to configure the scalar. Something like this https://www.apollographql.com/docs/apollo-server/schema/custom-scalars/#example-restricting-integers-to-odd-values

Reply:
maybe using this package would make it easier https://github.com/Urigo/graphql-scalars